modules/gitops-argo-cd-properties.adoc

```diff
@@ -51,6 +56,12 @@ a|* `<processors.operation>` - The number of operation processors.
   * `<env>` - Environment to set for the application controller workloads.
   * `<sourceNamespaces>` - The list of non-control plane namespaces for creating and managing Argo CD `Application` resources in target namespaces.
   * `<extraCommandArgs>` - List of arguments added to the existing arguments set by the Operator.
+  * `annotations` - The number of custom annotations that are added to pods deployed by the Operator.
+  * `labels` - The number of custom labels that are added to pods deployed by the Operator.
+  * `volumes` - The number of addition volumes configured for the Argo CD Application Controller component. This field is optional.
+  * `volumeMounts` - The number of addition volume mounts configured for the Argo CD Application Controller component. This field is optional.
```

Comment:
```suggestion
  * `volumes` - List of addition volumes configured for the Argo CD Application Controller component. This field is optional.
  * `volumeMounts` - List of addition volume mounts configured for the Argo CD Application Controller component. This field is optional.
```
Can we update with this for all volumes & volumeMounts properties in this PR?

Reply:
Fixed.